.travis.yml

```diff
@@ -111,6 +111,8 @@ matrix:
         - . ./ci/travis/ci.sh build
       script:
         - . ./ci/travis/ci.sh test_wheels
+        # Perform sanity check for docker build
+        - if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then ./ci/keep_alive bash $TRAVIS_BUILD_DIR/ci/travis/build-autoscaler-images.sh; fi
```

Comment:
Could this be `"$TRAVIS_PULL_REQUEST" == "true"?

Reply:
https://docs.travis-ci.com/user/environment-variables/:
> TRAVIS_PULL_REQUEST is set to the pull request number if the current job is a pull request build, or false if it’s not.

Reply:
Ahh okay, that makes sense!